src/components/AccessForm.vue

```diff
@@ -73,24 +80,50 @@ export default defineComponent({
             trigger: "change"
           }
         ]
-      }
+      },
+      loading: false
     };
   },
+  computed: {
+    ...mapState(["ourSecretKey", "executionToken"]),
+    ...mapGetters(["enclavePublicKey"])
```

Comment:
We can remove the unused accessors here.

```suggestion
    ...mapState(["executionToken"])
```